webextension/background/loadSelector.js

```diff
@@ -2,6 +2,8 @@
 
 window.loadSelector = (function () {
   const scripts = [
+    "background/loadSelector.js",
+    "selector/lifecycle.js",
```

Comment:
I notice that the order of this list matters. I think there ought to be a comment about that so someone doesn't reorder accidentally.